TextInputKit/TextInputKit/Code/SpecializedTextInput/BankCardHolderName/BankCardHolderNameTextInputOptions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  BankCardHolderNameTextInputOptions.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 15/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+/// Options for text input of bank card numbers.
+///
+/// - Note:
+///   Currently, there are no properties in `BankCardHolderNameTextInputOptions` struct.
+///   It is left for future text input customizations not breaking the API.
+public struct BankCardHolderNameTextInputOptions {
+
+    public static func options() -> BankCardHolderNameTextInputOptions {
+        return self.init()
```

Comment:
The idea in providing such "options" structures and `options` arguments in method signatures is to add new functionality/customizations seamlessly in future. It's not worth adding `.empty`; the intention is to add new arguments to `.options()` with default values.
"Why not to use `.init()`?" — to make code that uses such methods self-explaining, like `foo(.options(...))`.